From db1b25037e2cf1132dd8d0211d35407c5e68cb55 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 28 Aug 2018 09:54:02 +0800
Subject: [PATCH] dt-bindings: usb: dwc3: add grx/gtx-threshold-cfg properties

Add snps,grx-threshold-cfg and snps,gtx-threshold-cfg properties
which provide values for global Rx/Tx threshold control.

Change-Id: Id470cdc8a1bcf7fe8048ecaa040f5b5662b8ce05
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 Documentation/devicetree/bindings/usb/dwc3.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/dwc3.txt b/Documentation/devicetree/bindings/usb/dwc3.txt
index 408639c20c8c..31ddca7b41eb 100644
--- a/Documentation/devicetree/bindings/usb/dwc3.txt
+++ b/Documentation/devicetree/bindings/usb/dwc3.txt
@@ -60,6 +60,8 @@ Optional properties:
 	register for post-silicon frame length adjustment when the
 	fladj_30mhz_sdbnd signal is invalid or incorrect.
  - snps,usb3-warm-reset-on-resume-quirk: when set, need warm reset on resume.
+ - snps,grx-threshold-cfg: global Rx threshold config
+ - snps,gtx-threshold-cfg: global Tx threshold config
 
  - <DEPRECATED> tx-fifo-resize: determines if the FIFO *has* to be reallocated.
 
-- 
2.35.3

